core/osrd-geom/src/main/java/fr/sncf/osrd/geom/Point.java

```diff
@@ -10,15 +10,17 @@ public record Point(
 
     /**
      * Returns the distance between this point and another in meters, assuming x = longitude and y =
-     * latitude
+     * latitude. Uses equirectangular distance approximation (very fast but not 100% accurate)
```

Comment:
Do you know if the inaccuracies keep the heuristic admissible? Or are they small enough that they don't matter?

Reply:
No, the heuristic isn't admissible anymore. But I believe that it's small enough to not be an issue. Though if we really want to make it admissible again, we could scale the value down by like 1%.

But since we now look for the fastest path instead of shortest, there's a fair amount of approximations anyway. I don't think it would make sense to be too rigid on finding the single "best possible path"